app/main.py

```diff
@@ -1,6 +1,35 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    base_url = "https://api.weatherapi.com/v1/current.json?"
+    api_key = os.getenv("API_KEY")
+
+    city = "Seoul"
+
+    if not os.getenv("API_KEY"):
+        raise TypeError(
+            "Set your environment variable API_KEY "
+            "in terminal and run this script again"
+        )
+
+    url = base_url + "key=" + api_key + "&q=" + city
+
+    try:
+        requests.get(url).status_code == 200
+    except ValueError:
+        print("Invalid status code")
+
+    response_json = requests.get(url).json()
```

Comment:
```suggestion
    response = requests.get(url)
    
    if response.status_code != 200:
        raise ValueError("Invalid status code")

    response_json = requests.get(url).json()
```
== really can throw an error?